flatpak/io.github.celluloid_player.Celluloid.json

```diff
@@ -0,0 +1,188 @@
+            ]
+        },
+        {
+            "name": "ffmpeg",
+            "cleanup": [
+                "/include",
+                "/lib/pkgconfig",
+                "/share/ffmpeg/examples"
+            ],
+            "config-opts": [
+                "--enable-shared",
+                "--disable-static",
+                "--enable-gnutls",
+                "--disable-doc",
+                "--disable-programs",
+                "--disable-encoders",
+                "--disable-muxers",
+                "--enable-encoder=png",
+                "--enable-libv4l2"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://git.ffmpeg.org/ffmpeg.git"
+                }
+            ]
+        },
+        {
+            "name": "fribidi",
+            "cleanup": [
+                "/bin",
+                "/include",
+                "/lib/pkgconfig",
+                "/lib/*.la",
+                "/share/man"
+            ],
+            "buildsystem": "meson",
+            "config-opts": [ "-Ddocs=false" ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/fribidi/fribidi.git"
+                }
+            ]
+        },
+        {
+            "name": "libass",
+            "cleanup": [
+                "/include",
+                "/lib/*.la",
+                "/lib/pkgconfig"
+            ],
+            "config-opts": [
+                "--disable-static"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/libass/libass.git"
+                }
+            ]
+        },
+        {
+            "name": "uchardet",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DBUILD_STATIC=0"
+            ],
+            "cleanup": [
+                "/bin",
+                "/include",
+                "/lib/pkgconfig",
+                "/share/man"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.freedesktop.org/uchardet/uchardet.git"
+                }
+            ]
+        },
+        {
+            "name": "libmpv",
+            "cleanup": [
+                "/include",
+                "/lib/pkgconfig"
+            ],
+            "buildsystem": "simple",
+            "build-commands": [
+                "python3 waf configure --prefix=/app --enable-libmpv-shared --disable-cplayer --disable-build-date --disable-oss-audio --disable-alsa --enable-tv",
+                "python3 waf build",
+                "python3 waf install"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/mpv-player/mpv/archive/v0.29.1.tar.gz",
```

Comment:
If anything this is probably the most important dep to build from git?

Reply:
ATM if this is built from Git, the program crashes on just about any input.

Reply:
Fair enough.